trunk/cli/README.md

```diff
@@ -1,51 +1,82 @@
+The [Trunk PostgreSQL Extension Registry](https://pgtrunk.io), henceforth Trunk, offers a companion CLI (command line interface) to facilitate a user-friendly programmatic interface. This toolset, written in Rust, leverages the package manager, Cargo, and lowers the barriers to building, sharing, and using PostgreSQL extensions.
 
-# Introducing Trunk
+## Installation
 
-Trunk serves as registry where users can publish, search, and download community-made Postgres extensions. Inspired by popular developer hubs, such as [crates.io](http://crates.io) (Rust), [pypi.org](http://pypi.org) (Python), and [npmjs.com](http://npmjs.com) (JavaScript), Trunk aims to foster an information-rich environment. Here, developers can interact with the registry in a variety of ways and proudly showcase their contributions. Furthermore, users can gain insights into valuable metrics on extension downloads and trends.
+The Trunk CLI can be installed using the following command:
 
-At its core, the goal of trunk is to cultivate a thriving Postgres extension ecosystem by lowering the barriers to building, sharing, and using Postgres extensions.
+`cargo install pg-trunk`
 
-# Roadmap
+By default, the files are stored (). To confirm its proper installation, invoke the following:
 
-The Trunk infrastructure can be divided into the following: a command line interface (CLI), a registry, and a website.
+`trunk --version`
 
-### The CLI
+## Commands
 
 The CLI toolkit will abstract away many complexities in extension development and installation by using the following commands:
 
-`trunk init`
-- setup your environment to build a new Postrgres extension.
+- `trunk init` - setup your environment to build a new Postrgres extension.
+- `trunk test` - facilitate the automated unit and integration testing Postgres extensions.
+- `trunk build` - compiles extensions and supports nested dependencies.
+- `trunk publish` - publishes an extension to the registry, making it available for discovery and installation.
+- `trunk install` - download and install the extension distribution, in whichever environment trunk is run.
 
-`trunk test`
-- facilitate the automated unit and integration testing Postgres extensions.
+### 1. `trunk init`
 
-`trunk build`
-- compiles extensions.
-- supports nested dependencies, e.g. installing `extension_a` will automatically install `extension_b` if required.
+### 2. `trunk test`
 
-`trunk publish`
-- publishes an extension to the registry, making it available for discovery and installation.
+### 3. `trunk build`
+e.g. installing extension_a will automatically install extension_b if required
```

Comment:
I don't think this belongs here. This sounds like dependency management (part of `trunk install`)